examples/docs-examples/examples/selecting-providers/whitelist.mjs

```diff
@@ -1,31 +1,24 @@
 import { TaskExecutor, ProposalFilters } from "@golem-sdk/golem-js";
 
 /**
- * Example demonstrating how to use the predefined filter `whiteListProposalIdsFilter`,
- * which only allows offers from a provider whose ID is in the array
+ * Example demonstrating how to use the predefined filter `whiteListProposalNamesFilter`,
+ * which only allows offers from a provider whose name is in the array
  */
 
-const whiteListIds = [
-  "0x3a21c608925ddbc745afab6375d1f5e77283538e",
-  "0xd79f83f1108d1fcbe0cf57e13b452305eb38a325",
-  "0x677c5476f3b0e1f03d5c3abd2e2e2231e36fddde",
-  "0x06c03165aaa676680b9d02c1f3ee846c3806fec7",
-  "0x17ec8597ff92c3f44523bdc65bf0f1be632917ff", // goth provider-1:
-  "0x63fc2ad3d021a4d7e64323529a55a9442c444da0", // goth provider-2:
-];
+const whiteListNames = ["provider-2"];
```

Comment:
we should also include the providers that were there before, but we need to find their names in stats.golem.network, e.g. 0x3a21c608925ddbc745afab6375d1f5e77283538e -> fractal_01_3.h

If someone copies this example from our documentation and wants to run it on their machine, they will not find provider-2

Reply:
Good point. Will address